docs/source/operations/operations_computation.rst

```diff
@@ -0,0 +1,741 @@
+                   +step +proj=push +v_3 \
+                   +step +proj=cart +ellps=aust_SA \
+                   +step +proj=helmert +x=-117.763 +y=-51.51 +z=139.061 \
+                                       +rx=-0.292 +ry=-0.443 +rz=-0.277 +s=-0.191 \
+                                       +convention=coordinate_frame \
+                   +step +proj=helmert +x=0.06155 +y=-0.01087 +z=-0.04019 \
+                                       +rx=-0.0394924 +ry=-0.0327221 +rz=-0.0328979 \
+                                       +s=-0.009994 +convention=coordinate_frame \
+                   +step +inv +proj=cart +ellps=GRS80 \
+                   +step +proj=pop +v_3 \
+                   +step +proj=unitconvert +xy_in=rad +xy_out=deg \
+                   +step +proj=axisswap +order=2,1
+
+    -------------------------------------
+    Operation n°3:
+
+    unknown id, AGD84 to GDA94 (5) + GDA94 to GDA2020 (2), 0.15 m, unknown domain of validity
+
+    PROJ string:
+    +proj=pipeline +step +proj=axisswap +order=2,1 \
+                   +step +proj=unitconvert +xy_in=deg +xy_out=rad \
+                   +step +proj=hgridshift +grids=National_84_02_07_01.gsb \
+                   +step +proj=hgridshift +grids=GDA94_GDA2020_conformal_and_distortion.gsb \
+                   +step +proj=unitconvert +xy_in=rad +xy_out=deg \
+                   +step +proj=axisswap +order=2,1
+
+    -------------------------------------
+    Operation n°4:
+
+    unknown id, AGD84 to GDA94 (5) + GDA94 to GDA2020 (3), 0.15 m, unknown domain of validity
+
+    PROJ string:
+    +proj=pipeline +step +proj=axisswap +order=2,1 \
+                   +step +proj=unitconvert +xy_in=deg +xy_out=rad \
+                   +step +proj=hgridshift +grids=National_84_02_07_01.gsb \
+                   +step +proj=hgridshift +grids=GDA94_GDA2020_conformal.gsb \
+                   +step +proj=unitconvert +xy_in=rad +xy_out=deg \
+                   +step +proj=axisswap +order=2,1
+
+One can see that the selected intermediate CRS that has been used is GDA94.
+This is a completely novel behaviour of PROJ 6 regarding to the PROJ.4 logic
+where datum transformations implied using EPSG:4326 / WGS 84 has the mandatory
+datum hub. PROJ 6 no longer hardcodes it as the mandatory datum hub, and relies
+on the database to find the appropriate hub(s).
+Actually, WGS 84 has been considered during the above lookup, because there are
+transformations between AGD84 and WGS 84 and WGS 84 and GDA2020. However those
+have been discarded in a step which we did not mention previously: just after
+the initial filtering of results and their sorting, there is a final filtering
+that is done. In the list of sorted results, if a less prioritary result than
+its previous one has the same area of use, but a lesser accuracy and that the
+more accurace results does not use grids, or the grids are available, then the
+less accurate result is discarded.
```

Comment:
Try to rephrase this sentence. I struggle to figure out what you are saying in the beginning. If all other things are equal, the less accurate hub datum is discarded, I guess?

Reply:
rephrased as "In the list of sorted results, given two operations A and B that
have the same area of use, if B has an accuracy lower than A, and A does not use
grids, or all the needed grids are available, then B is discarded."